funtests/transport.py

```diff
@@ -21,13 +21,19 @@
 else:
     from sha import new as _digest  # noqa
 
+if not hasattr(string, 'letters'):
```

Comment:
Ah, so it was removed in Python 3!  Seems the reasoning behind the change is that `string.letters` actually varies defined on the locale set (which is news to me), so string.ascii_letters is probably what we should use always.